stdlib/inspect.pyi

```diff
@@ -154,16 +154,23 @@ if sys.version_info >= (3, 10):
         eval_str: bool = ...,
     ) -> Dict[str, Any]: ...
 
-# The name is the same as the enum's name in CPython
-class _ParameterKind(enum.IntEnum):
-    POSITIONAL_ONLY: int
-    POSITIONAL_OR_KEYWORD: int
-    VAR_POSITIONAL: int
-    KEYWORD_ONLY: int
-    VAR_KEYWORD: int
-
-    if sys.version_info >= (3, 8):
-        description: str
+if sys.version_info >= (3, 0):
+    import enum
+
+    # The name is the same as the enum's name in CPython
+    class _ParameterKind(enum.IntEnum):
+        POSITIONAL_ONLY: int
+        POSITIONAL_OR_KEYWORD: int
+        VAR_POSITIONAL: int
+        KEYWORD_ONLY: int
+        VAR_KEYWORD: int
+
+        if sys.version_info >= (3, 8):
+            description: str
+
+else:
+    # TODO: This actually doesn't exist on Python 2
+    _ParameterKind = int
 
 class Parameter:
```

Comment:
I think Parameter and BoundArguments don't exist in Python 2 either, could you move them within the Python 3 block?

Reply:
Yeah. I moved some Python 3 only definitions behind version checks.